.github/workflows/e2e.ci.yaml

```diff
@@ -43,98 +46,29 @@ jobs:
           repository: apache/skywalking
           submodules: true
           path: main
-          ref: 6228d450e1b0792dd9875ad68984c89c81ece742
+          ref: 8436135dc919351a6d0ab42ff48c85e1b3104b53
```

Comment:
I think this entire `actions/checkout` step can be removed?

Reply:
It's could not remove, the E2E need to reference the cases excepted, environment files.

Reply:
OK if that's the case, it means you cannot run the tests locally without manually cloning/copying the main repo to a specific location, I'd rather copy those files or put the `checkout` step into the `e2e.yaml`.

Also, you cannot simply use `OAP_TAG: latest` (line 30) and `UI_TAG: latest` (line 32) when you use a specific revision of the expected files, they might break in the future.

Reply:
What do you mean about the checkout? checkout the main repo into the `setup.steps`?

Reply:
yes, checkout the main repo in `setup.steps`.

But I prefer to copy those files into this repo and don't depend on that repo as these files are not strong relevant to the main repo